lib/renderers/DefaultRenderer.js

```diff
@@ -68,7 +68,13 @@ class DefaultRenderer {
   }
 
   renderMessage(msg, event) {
-    function deepSearchStatement(statement, alexaEvent) {
+    /**
+     * It searchs deep looking for string that could have a variable on it
+     * @param  any statement
```

Comment:
perhaps an example of what a statement looks like and how it would look like after rendering